fabric-containers-v0/src/main/java/net/fabricmc/fabric/impl/container/ContainerProviderImpl.java

```diff
@@ -91,7 +91,7 @@ public void openContainer(Identifier identifier, ServerPlayerEntity player, Cons
 		buf.writeByte(syncId);
 
 		writer.accept(buf);
-		player.networkHandler.sendPacket(new CustomPayloadS2CPacket(PacketTypes.OPEN_CONTAINER, buf));
+		player.networkHandler.sendPacket(new CustomPayloadS2CPacket(OPEN_CONTAINER, buf));
```

Comment:
I know its not touched by you PR atm, but what about an easier way to create the packet? I know some people find it confusing to create the `PacketByteBuf` along with the correct buffer?

Reply:
First, good point. I should change this to `PacketSenders.of(player.networkHandler).send(OPEN_CONTAINER, buf)`

Second, though I have documented how we can create `PacketByteBuf` instances, should I add a utility method like in `PacketSenders`?